canine/localization/nfs.py

```diff
@@ -116,6 +108,20 @@ def localize(self, inputs: typing.Dict[str, typing.Dict[str, str]], patterns: ty
         """
         if overrides is None:
             overrides = {}
+
+        # automatically override inputs that are absolute paths residing on the same
+        # NFS share and are not Canine outputs
+
+        # XXX: this can be potentially slow, since it has to iterate over every
+        #      single input. It would make more sense to do this before the adapter
+        #      converts raw inputs.
```

Comment:
Here is the adapter vs. localizer issue we discussed last week.

Reply:
I think I'll tackle this separately in #25